frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -460,12 +461,18 @@
    "fieldname": "prepared_report_section",
    "fieldtype": "Section Break",
    "label": "Prepared Report"
+  },
+  {
+   "default": "1",
+   "fieldname": "remove_exif_tags",
+   "fieldtype": "Check",
+   "label": "Remove EXIF tags from uploaded images"
```

Comment:
```suggestion
   "fieldname": "strip_exif_metadata_from_uploaded_images",
   "fieldtype": "Check",
   "label": "Strip EXIF tags from uploaded images"
```

This fieldname seems more appropriate when you think of all images just data formatted with EXIF. Also, since we're only implementing EXIF stripping for image file specifications (excluding audio).

Reply:
cool, will change.